src/arkode/arkode_arkstep.c

```diff
@@ -2212,14 +2200,14 @@ int arkStep_AccessStepMem(void* arkode_mem, const char* fname,
   /* access ARKodeMem structure */
   if (arkode_mem == NULL)
   {
-    arkProcessError(NULL, ARK_MEM_NULL, __LINE__, __func__, __FILE__,
+    arkProcessError(NULL, ARK_MEM_NULL, __LINE__, fname, __FILE__,
```

Comment:
I think a similar update is missing from `arkLs_AccessLMem`, `arkLs_AccessMassMem`, and `arkRelaxAccessMem`